docs/how-to/opentelemetry-migration.md

```diff
@@ -0,0 +1,41 @@
+(opentelemetry-migration)=
+# Migrate from Prometheus/Grafana to OpenTelemetry/Prometheus/Grafana 
+
+The {ref}`Prometheus/Grafana <monitoring>`based monitoring setup will soon be deprecated in favor of the {ref}`OpenTelemetry setup <opentelemetry>`. This section provides the details required to update/migrate your Prometheus configuration and Grafana dashboard to continue monitoring with OpenTelemetry. Refer to {ref}`Opentelemetry setup <opentelemetry>` for the new setup before proceeding further.
+
+```{hint}
+:class: seealso
+Refer to {ref}`Prometheus/Grafana-only <monitoring>` section for the soon to be deprecated setup.
+```
+
+## Update Prometheus configuration
+
+With a Prometheus-only setup, a `scrape_configs` configuration or a service discovery plugin is needed to be configured to specify the targets to pull metrics data. In the OpenTelemetry setup, each Pod pushes the metrics to the OpenTelemetry Collector. The Prometheus configuration now only needs to scrape from the OpenTelemetry Collector to get all the data from OpenTelemetry instrumented applications.
+
+The new Prometheus configuration for the `otel-collector` Collector hostname is:
+
+```yaml
+scrape_configs:
+  - job_name: 'otel-collector'
+    scrape_interval: 500ms
+    static_configs:
+      - targets: ['otel-collector:8888'] # metrics from the collector itself
+      - targets: ['otel-collector:8889'] # metrics collected from other applications
+```
+
+## Update Grafana dashboard
+
+The OpenTelemetry [Histogram](https://opentelemetry.io/docs/reference/specification/metrics/api/#histogram) provides quantile window buckets automatically unlike the Prometheus [Summary](https://prometheus.io/docs/concepts/metric_types/#summary) instrument. The required quantile window needed to be manually configured. This would then be available as a separate time series metric.
```

Comment:
```suggestion
The OpenTelemetry [Histogram](https://opentelemetry.io/docs/reference/specification/metrics/api/#histogram) provides quantile window buckets automatically (unlike the Prometheus [Summary](https://prometheus.io/docs/concepts/metric_types/#summary) instrument.) You need to manually configure the required quantile window. This will then be available as a separate time series metric.
```

What's "This" in the last sentence?